### PR TITLE
feat: add network encryption recommender

### PR DESCRIPTION
**Pull request by galexrt**
This adds an optional recommendation to enable cluster network encryption.

image

### CODE REVIEW

File: pkg/recommender/modules/rook_encryption.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	for _, item := range clusters.Items {
+		if item.Spec.Network.Connections.Encryption != nil && item.Spec.Network.Connections.Encryption.Enabled {
+			continue
+		}
+
+		recs = append(recs, &cephv1.ClusterRecommendation{
+			Title: "Ceph network encryption not active",
+			Description: `It is recommended to enable Ceph network encryption for clusters that might be running in \"untrusted\" networks.
+To enable it carefully read the \"encryption.enabled\" settings comments <a class=\"underline\" href=\"https://rook.io/docs/rook/latest-release/CRDs/Cluster/ceph-cluster-crd/#network-configuration-settings\">here</a>.`,
+			Level: cephv1.RecommendationLevel_RECOMMENDATION_LEVEL_INFORMAL,
```
Comment (zalsader) [inline pkg/recommender/modules/rook_encryption.go:42]: On the name of this, does "informal" mean "not formal" or "providing information, informational"? If it's the latter, I suggest renaming this to `info`.

**Review by zalsader — APPROVED** [review]
LGTM

File: pkg/recommender/modules/rook_encryption.go (hunk 1 above)
Comment (galexrt) [reply pkg/recommender/modules/rook_encryption.go:42]: I have renamed it to `info`/`INFO`, merging now.

Comment (zalsader) [reply pkg/recommender/modules/rook_encryption.go:42]: I do not see the change merged, did you push it?

**Pull request merged**